SPI - OpenMV H7 ESP 32 Interface/ESP32 Interface Code/ESP32_Interface_Code/ESP32_Interface_Code.ino

```diff
@@ -0,0 +1,61 @@
+#include <SPI.h>
+
+#define CHAR_BUF 128
+#define SS 4
+#define MISO 19 
+#define MOSI 18
+#define SCLK 5
+
+static const int spiClk = 1;
+
+SPIClass * hspi = NULL;
+
+void setup() {
+  
+  Serial.begin(115200);
+
+  hspi = new SPIClass(HSPI);
+  pinMode(SCLK, OUTPUT);
+  pinMode(MISO, INPUT);
+  pinMode(MOSI, OUTPUT);
+  pinMode(SS, OUTPUT);
+  
+  hspi->begin(SCLK, MISO, MOSI, SS);
+
+  hspi->beginTransaction(SPISettings(spiClk, MSBFIRST, SPI_MODE0));
+  
+}
+
+void loop() {
+  uint8_t len = 0;
+  uint32_t len_temp = 0;
+  uint8_t buff2[CHAR_BUF] = {0};
+
+  digitalWrite(SS, LOW);
+  Serial.println("Requesting data.....");
+  delay(1); // Give the OpenMV Cam some time to setup to send data.
+
+  if(hspi->transfer(1) == 85) { // saw sync char?
```

Comment:
85 = 0x54
Will not be met under normal operation
0x54XX
is the currently expected output